Entities/Structures/Trampoline/TrampolineLogic.as

```diff
@@ -41,12 +42,20 @@ void onTick(CBlob@ this)
 
 	Vec2f ray = holder.getAimPos() - this.getPosition();
 	ray.Normalize();
-
+	
 	f32 angle = ray.Angle();
-	angle = angle > 135 || angle < 45? (holder.isFacingLeft()? 135 : 45) : 90;
-	angle -= 90;
+	CControls@ controls = getControls();
 
-	this.setAngleDegrees(-angle);
+	if (controls.isKeyPressed(KEY_LCONTROL))
```

Comment:
Any reason why you chose ctrl over shift?
I think you should use `AK_BUILD_MODIFIER` which is bound to ctrl by default and is bindable.
```suggestion
	if (controls.isKeyPressed(AK_BUILD_MODIFIER))
```